crates/notan_egui/src/plugin.rs

```diff
@@ -44,13 +44,14 @@ impl EguiPlugin {
 
         let egui::FullOutput {
             platform_output,
-            repaint_after,
             textures_delta,
             shapes,
+            pixels_per_point,
+            viewport_output,
         } = self.ctx.run(new_input, run_ui);
 
         let needs_update_textures = !textures_delta.is_empty();
-        let needs_repaint = repaint_after.is_zero() || needs_update_textures;
+        let needs_repaint = viewport_output.values().any(|output| output.repaint_delay.is_zero()) || needs_update_textures;
```

Comment:
This is my best guess. `FullOutput::repaint_after` was removed in egui#3172 and I'm not sure how dealing with multiple viewports effects notan.